communication/include/communication/CommDefs.hpp

```diff
@@ -150,13 +160,17 @@ class TlsTcpConfig : public PlainTcpConfig {
         useUnifiedCertificates_{useUnifiedCerts},
         secretData_{std::move(decryptionSecretData)} {
     commType_ = CommType::TlsTcp;
+    if (tcpKeepAliveConfig != nullptr) {
+      tcpKeepAliveConfig_ = *tcpKeepAliveConfig;
+    }
   }
 
  public:
   std::string certificatesRootPath_;
   std::string cipherSuite_;
   bool useUnifiedCertificates_;
   std::optional<concord::secretsmanager::SecretData> secretData_;
+  TcpKeepAliveConfig tcpKeepAliveConfig_;
```

Comment:
It should be  a base class `PlainTcpConfig` configuration property

Reply:
I thought about it. PlainTcpConfig is never used, but I still can move the property there.